cypress/integration/examples/app.spec.js

```diff
@@ -1,23 +1,176 @@
+            .get('.values-button').click()
+              .get('.account-values-text').should('have.text', "Food and Stuff")
+              cy.get('.values-info').should('have.text', "Food and Stuff")
+  })
+
+  it('Should have an about page', () =>{
+    cy.get('.about-link').click()
+      cy.url().should('contain', '/about')
+      cy.get('.about').children('.about-title', '.about-info')
+        .get('.about-title').should('have.text', 'About PD-Pal')
+        .get('.about-info').should('contain', 'This was the final solo project for Mod 3.').children('.alice-link', '.turing-link')
+          .get('.alice-link').should('have.attr', 'href', 'https://github.com/srslie')
+          .get('.turing-link').should('have.attr', 'href', 'https://turing.io' )
+  })
+
+  it('Should have a page with resources for PD', () =>{
+    cy.get('.resources-link').click()
+      cy.url().should('contain', '/resources')
+      cy.get('.resources').children('.resources-title', '.resources-info', '.pd-link')
+        .get('.resources-title').should('have.text', 'Professional Development Resources')
+        .get('.resources-info').should('contain', 'Turing School empahasizes professional development.').children('.pd-link').should('have.attr', 'href', 'https://github.com/turingschool/career-development-curriculum')
+          .get('.pd-link-button').should('have.text', 'Find out more!')
+  })
+
+  it('Should allow users to see saved jobs', () => {
+    cy.get('.details-link:first').click()
+      .get('.user-interactions')
+        .get('.save-button').should('have.text', 'Save').click()
+        .get('.saved-link').click()
+          cy.url().should('contain', '/saved')
+          .get('.job-card').should('have.id', '2035955654')
+  })
+
+  it('Should allow users to see jobs they applied to', () =>{
+    cy.get('.details-link:first').click()
+      .get('.user-interactions')
+        .get('.applied-button').should('have.text', 'Mark Applied').click()
+        .get('.applied-link').click()
+          cy.url().should('contain', '/applied')
+          .get('.job-card').should('have.id', '2035955654')
+  })
+
   it('Should redirect to a 404 page for unknown urls', () =>{
-    cy.visit('http://localhost3000/sheep')
+    cy.visit('http://localhost:3000/sheep')
+      cy.get('.not-found').children('.not-found-overlay')
+        .get('.not-found-overlay').children('.not-found-code', '.not-found-title', '.home-link')
+        .get('.not-found-code').should('have.text', '404:')
+        .get('.not-found-title').should('have.text', 'Sorry, page not found.')
+        .get('.home-link').children('.home-link-button')
+        .get('.home-link-button').click()
+      cy.url().should('eq', 'http://localhost:3000/')
   })
-});
\ No newline at end of file
+})
+
+describe('Failure', () => {
+  it.only('Should show an error page if fetch fails', () => {
+    cy.intercept(
+      'GET', 
+      apiUrl,
+      { statusCode: 400}
+    )
```

Comment:
Could be worth testing at least one other kind of error

Reply:
Good point! Added!